### ISSUE TITLE
Rounding issues with sequence

### ISSUE THREAD

**Issue by JonnyBurger** [remotion-dev/remotion#2958]
This unexpectedly gives an empty frame: 

```tsx
import {Sequence, AbsoluteFill, useVideoConfig} from 'remotion';

export const HelloWorld: React.FC = () => {
	return (
		<AbsoluteFill style={{backgroundColor: 'white'}}>
			<Sequence from={5.1000000000000005} durationInFrames={128.4}>
						<h1>One</h1>
					</Sequence>
					<Sequence from={133.5} durationInFrames={96.72}>
						<h1>Two</h1>
					</Sequence>
		</AbsoluteFill>
	);
};
```

**Comment by JonnyBurger** [comment]
Fixed by #2961

**Comment by Quentintin** [comment]
I fixed it manually with Math.ceil the previous week, but you finally fix it ;) 

Next time, I will push a PR for that.
